app/src/main/java/com/skooterapp/ComposeActivity.java

```diff
@@ -422,7 +422,7 @@ public void main() throws Exception {
             ByteArrayOutputStream stream = new ByteArrayOutputStream();
 
             ByteArrayBody byteArrayBody;
-            ArrayList<NameValuePair> nameValuePairs = new ArrayList<NameValuePair>();
+//            ArrayList<NameValuePair> nameValuePairs = new ArrayList<NameValuePair>();
```

Comment:
This ArrayList is not used anymore, It was referenced in line (578) but this line was commented out  and nothing is referencing it anymore and it's considered a dead store.